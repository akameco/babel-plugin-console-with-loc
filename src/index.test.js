// @flow
import pluginTester from 'babel-plugin-tester'
import plugin from '.'

pluginTester({
  title: 'transform console.log',
  plugin,
  snapshot: true,
  tests: [
    {
      title: 'single console.log',
      code: `
    console.log('hello world')
    `,
    },
    {
      title: 'no console.log',
      code: `
    log('hello world')
    `,
    },
  ],
})
