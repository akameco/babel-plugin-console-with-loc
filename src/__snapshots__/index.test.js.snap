// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no console.log 1`] = `
"
log('hello world')

      ↓ ↓ ↓ ↓ ↓ ↓

log('hello world');
"
`;

exports[`single console.log 1`] = `
"
console.log('hello world')

      ↓ ↓ ↓ ↓ ↓ ↓

console.log('{\\"start\\":{\\"line\\":1,\\"column\\":0},\\"end\\":{\\"line\\":1,\\"column\\":26}}', 'hello world');
"
`;
